### ISSUE TITLE
Ver 2.19 Missing Read Receipt Timestamps

### ISSUE THREAD

**Issue by redstig** [signalapp/Signal-iOS#2817]
I am using iPhone 7 Plus and the Signal has the latest version (2.19.0). The conversation stopped showing the timestamps when the read receipt is on. Desperately need the feature back. 

Thank you.

**Comment by michaelkirk** [comment]
Have you verified that both you and the person you're communicating with had enabled read receipts at the time the message was first read?

Also - please don't delete the issue template - it slows down this whole process because now I have to ask you for the same things manually.

What client/version was the other person using when this occurred?

Please provide a debug log from your client after observing the behavior 
Please provide a debug log from their client during the same time frame.

**Comment by redstig** [comment]
Hi Michael,

I am really sorry for deleting the issue template. The template was confusing since I posted the thread via my iPhone. Sorry for the inconvenience.

Yes, I have verified that both you and the person I was communicating with had enabled read receipts at the time the message was first read.

Both of us are using IOS version 2.19.0.22, specifically I am using iPhone 7 Plus. The person I was communicating has iPhone X.

Debug log from my client: https://gist.github.com/f0adc0fdfa430944e0524f11210ca470
Debug log from the person I talked with: https://gist.github.com/1a996585bb1e4068665adc6f01af608c

Edit: I noticed that the timestamp functionality still exists. However, I have to manually hold the message box for 2 seconds and click on the "Info" tab to retrieve the timestamp. In the previous Signal version (2.18.x.xx), I was able to directly see the timestamp right after the "Read" notice. I was wondering if there is anyway to see the timestamp directly without clicking into the "Info" tab?

Here is how the Read Receipt interface looks like in previous/current version of Signal:
Previous Signal Version: "Read 6:00PM"
Current Signal Version: "Read"
